### DISCUSSION TITLE
Where to get ORG ID and PROJECT ID?

### DISCUSSION THREAD

**Discussion by brettski** [vercel/community#3307] [Help] [answered]
I am trying to set these two values for a project: `VERCEL_ORG_ID` and `VERCEL_PROJECT_ID`. Where are these values found? I tried the names from the vercel.com site, but they did not work resulting in a message of, "Error! Project not found"

**Comment by mcsdevv** [comment]
Hey @brettski, to use these you can follow the documentation here: https://vercel.com/docs/v2/build-step#system-environment-variables

Please let me know if there's anything else you'd like to clarify.

**Reply by brettski** [reply]
Thank you. I am aware of **how** to add environmen variables. What I am looking for is **_what_** is the value which needs to be used for `VERCEL_ORG_ID` AND `VERCEL_PROJECT_ID`. From what I have discovered is that these need to be actual ID values, not the friendly text values (e.g. my-project-name) shown in the dashboard. How do I get the the actual id values? Thank you.

**Comment by cdock1029** [comment]
Check the ajax calls on https://vercel.com/dashboard/projects, there are some "ids" there. I'm guessing you can't explicitly set these anywhere, they are done so internally.

**Comment by willemliu** [comment]
When you connected your project to Vercel. By running `now`. You can find these values in the `.now` folder or is it now called `.vercel`?

**Comment by makoto** [comment]
In my case I am going to upgrading from `now` to handle multiple environments. I can find out project id of the one I linked on `.vercel/project.json` but wouldn't be able to know the project id for other environments.

**Comment by angusryer** [comment, marked as answer]
Just an update to this for reference:

You can find the org and project IDs in the project.json file in the `.vercel` folder of your local project.

OR

Your "Org ID" is also called your user ID, and it's found at [vercel.com/account](https://vercel.com/account) > settings > general -- at the bottom
Your "Project ID" is found at [vercel.com/dashboard](https://vercel.com/dashboard) > your project name > settings > general -- at the bottom

**Reply by dumez-k** [reply]
To add on to this, as I just discovered if your vercel project is managed by a Team the "Org ID" the CLI requires is actually the Team ID located: 

vercel.com/teams/YOUR_TEAM_NAME/settings > general under "Team ID"

**Reply by shoaib9121** [reply]
@angusryer  
Answer, Up to the mark. Bingo! (y)
